src/main/java/io/fabric8/maven/docker/config/handler/compose/DockerComposeConfiguration.java

```diff
@@ -7,12 +7,14 @@ public class DockerComposeConfiguration {
     private final String basedir;
     private final String composeFile;
     private final boolean ignoreBuild;
+    
     public DockerComposeConfiguration(Map<String, String> config) {
-        basedir = config.containsKey("basedir") ? config.get("basedir") : "src/main/docker";
-        composeFile = config.containsKey("composeFile") ? config.get("composeFile") : "docker-compose.yml";
-        ignoreBuild = config.containsKey("ignoreBuild") ? Boolean.parseBoolean(config.get("ignoreBuilder")) : false;
+    basedir = config.getOrDefault("basedir", "src/main/docker");
+    composeFile = config.getOrDefault("composeFile", "docker-compose.yml");
+    ignoreBuild = Boolean.parseBoolean(config.getOrDefault("ignoreBuild", "false"));
```

Comment:
indentation slightly off here

Reply:
Okay I'll fix the indentation and push again